include/fizzy/fizzy.h

```diff
@@ -157,6 +157,17 @@ const FizzyModule* fizzy_parse(const uint8_t* wasm_binary, size_t wasm_binary_si
 /// If passed pointer is NULL, has no effect.
 void fizzy_free_module(const FizzyModule* module);
 
+/// Make a copy of a module.
+///
+/// @param  module          Pointer to module. Cannot be NULL.
+/// @returns                Pointer to a newly allocated module, identical to @a module,
+///                         or NULL in case memory for a module couldn't be allocated.
+///
+/// @note Creating a copy is needed if more than single instance of a module is required, because
+/// instantiation takes ownership of a module, and the same module can't be instantiated twice.
```

Comment:
```suggestion
/// instantiation takes ownership of a module, and the same module cannot be instantiated twice.
```

We have not been using contractions in docs.

Reply:
Changed.